src/java/fr/paris/lutece/plugins/newsletter/web/portlet/NewsLetterArchivePortletJspBean.java

```diff
@@ -91,10 +95,22 @@ public String getCreate( HttpServletRequest request )
     {
         String strPageId = request.getParameter( PARAMETER_PAGE_ID );
         String strPortletTypeId = request.getParameter( PARAMETER_PORTLET_TYPE_ID );
-
-        HtmlTemplate template = getCreateTemplate( strPageId, strPortletTypeId );
-
-        return template.getHtml( );
+        Plugin plugin = PluginService.getPlugin( NewsLetterConstants.PLUGIN_NAME );
+        ArrayList<Integer> selectedSendings = new ArrayList<>();
+        List<SendingNewsLetter> sendingNewsletterList = SendingNewsLetterHome.findAllSendings( plugin );
+        HashMap<String, Object> model = new HashMap<String, Object>( );
+        model.put( MARK_SENDING_NEWSLETTER_LIST, sendingNewsletterList );
+        model.put( MARK_SELECTED_SENDING_LIST, selectedSendings );
+        HtmlTemplate templateCreate = getCreateTemplate( strPageId, strPortletTypeId );
+        HtmlTemplate templateNewsletterList = AppTemplateService.getTemplate(NewsLetterConstants.TEMPLATE_NEWSLETTER_SUBSCRIPTION_LIST, this.getLocale(), model);
+        // add the list of newsletterArchives to the create the template

```

Comment:
why use Jsoup rather than an include in the freemarker template ?
this would allow the java code to remain independent of the template.

Reply:
overiding create_portlet.html instead